### PR TITLE
Wordle with Optimint

### PR DESCRIPTION
**Pull request by YazzyYaz**
Optimint implementation of the Wordle cosmos-sdk app

Closes https://github.com/celestiaorg/docs/issues/156

### CODE REVIEW

**Review by adlerjohn — APPROVED** [review]
utACK

File: docs/developers/wordle-keeper.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+```
+
+Next file is `x/wordle/types/expected_keepers.go`
+
+Here, we need to add the SendCoins method to the BankKeeper
+interface in order to allow sending the reward to the right guesser.
+
+```go
+type BankKeeper interface {
+  SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
```
Comment (dabit3) [inline docs/developers/wordle-keeper.md:242]: There is a `SpendableCoins` method already there, do we remove this or keep both?

Comment (YazzyYaz) [reply docs/developers/wordle-keeper.md:242]: Yeah we replace it with `SendCoins`

File: docs/developers/run-wordle.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+the name `wordled`
+
+When the compilation finishes, it's time to start `wordled`. You
+can start the chain with optimint configurations by running the following:
+
+```sh
+wordled start --optimint.aggregator true --optimint.da_layer celestia --optimint.da_config='{"base_url":"http://XXX.XXX.XXX.XXX:26658","timeout":60000000000,"gas_limit":6000000,"namespace_id":[0,0,0,0,0,0,255,255]}' --optimint.namespace_id 000000000000FFFF --optimint.da_start_height 21380
+```
+
+> NOTE: In the above command, you need to pass a Celestia Node IP address
```
[the remark this answers is not in the document]
Comment (YazzyYaz) [reply docs/developers/run-wordle.md:24]: Hi @dabit3 so the `NOTE` section has a link to running a Celestia Light node as mentioned at the end:
```
the tutorial for setting up a Celestia Light Node and creating a wallet
with testnet faucet money [here](./node-tutorial.md) in the Celestia Node section.
```

Is that what you wanted or you are referring to something else?

File: docs/developers/wordle.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+## Ignite
+
+Ignite is an amazing CLI tool to help us get started building
+our own blockchains for cosmos-sdk apps. It provides lots of
+power toolings and scaffoldings for adding messages, types,
+modules with a host of cosmos-sdk libraries provided.
+
+You can read more about Ignite [here](https://docs.ignite.com/).
+
+To install Ignite, you can run this command in your Terminal:
```
Comment (dabit3) [inline docs/developers/wordle.md:73]: I see Terminal is capitalized sometimes but not other times.

File: docs/developers/scaffold-wordle.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Scaffolding the Wordle Chain
+<!-- markdownlint-disable MD013 -->
+
+Now, comes the fun part, creating a new blockchain! With Ignite,
+the process is pretty easy and straightforward.
+
+Ignite CLI comes with several of scaffolding commands that are
```
Comment (dabit3) [inline docs/developers/scaffold-wordle.md:7]: "several of scaffolding commands" -> "several several scaffolding commands"

File: docs/developers/scaffold-wordle.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+designed to make development more straightforward by creating
+everything you need to build your blockchain.
+
+First, we ill use Ignite CLI to build the foundation of a fresh
```
Comment (dabit3) [inline docs/developers/scaffold-wordle.md:11]: "we ill" -> "we will" or "we'll"

File: docs/developers/scaffold-wordle.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+Cosmos SDK blockchain. Ignite minimizes how much blockchain code
+must write yourself. If you are coming from the EVM-world, think of
+Ignite as a Cosmos-SDK version of Foundry or Hardhat but specifically
+designed to build blockchains.
+
```
Comment (dabit3) [inline docs/developers/scaffold-wordle.md:16]: "how much blockchain code must write yourself" -> "how much blockchain code you must write yourself"

**Review by tzdybal — APPROVED** [review]
LGTM

**Pull request merged**